src/server/pbsd_init.c

```diff
@@ -256,14 +256,18 @@ init_server_attrs()
 {
 	resource_def    *prdef = NULL;
 	resource        *presc = NULL;
-	struct attribute attrib = {0};
-	int i = 0;
+	attribute 	temp_attr;
+	struct 		attribute attrib = {0};
+	int 		i = 0;
+
 
 	for (i=0; i<SRV_ATR_LAST; i++)
 		clear_attr(&server.sv_attr[i], &svr_attr_def[i]);
 
-	server.sv_attr[(int)SRV_ATR_scheduler_iteration].at_val.at_long =
-		PBS_SCHEDULE_CYCLE;
+	temp_attr.at_flags = ATR_VFLAG_SET;
+	temp_attr.at_val.at_long = PBS_SCHEDULE_CYCLE;
```

Comment:
Yes @bhroam, we need to do clear_attr() on temp_attr. Since I have now created a new function which is used to set a value for any attribute from server I am taking care of calling this in that function set_attr_svr().